js/index.js

```diff
@@ -40,3 +40,70 @@ const siteContent = {
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
 logo.setAttribute('src', siteContent["nav"]["img-src"])
+
+
+let navAnchors = document.querySelectorAll('a');
+let mainHeader = document.querySelector('h1');
+let headerButton = document.querySelector('button');
+let headerImg = document.getElementById('cta-img');
+let H4 = document.querySelectorAll("h4");
+let paragraphs = document.querySelectorAll("p")
+let middleImg = document.getElementById("middle-img");
+
+
+let navList = document.querySelectorAll('nav a')
+for (let i = 0; i < navList.length; i++) {
+  navList[i].textContent = siteContent['nav'][`nav-item-${i + 1}`];
+  navList[i].style.color = 'green';
+};
+
+// new content added using prepend and append
+const newNavList = document.querySelector('nav');
+const navHome = document.createElement('a');
+const navBlog = document.createElement('a');
+navHome.href = '#';
+navHome.textContent = 'Home';
+navHome.style.color = 'green';
+navBlog.href = '#';
+navBlog.textContent = 'Blog';
+navBlog.style.color = 'green';
+
+newNavList.appendChild(navBlog);
+newNavList.prepend(navHome);
+
+//Header
+mainHeader.textContent = siteContent["cta"]["h1"];
+
+headerImg.setAttribute('src', siteContent["cta"]["img-src"]);
+headerButton.textContent = siteContent["cta"]["button"];
+
+H4[0].textContent = siteContent["main-content"]["features-h4"];
+H4[1].textContent = siteContent["main-content"]["about-h4"];
+H4[2].textContent = siteContent["main-content"]["services-h4"];
+H4[3].textContent = siteContent["main-content"]["product-h4"];
+H4[4].textContent = siteContent["main-content"]["vision-h4"];
```

Comment:
Oooh, we're repeating code here. What if we had a hundred main content elements, that'd be a lot of assignments to type out. So how could we make it more DRY?

Well, I see an opportunity here for loops! We just need to make sure that we're referring to the correct `siteContent` values. If we check out siteContent, we can see that the order in which our `-h4` keys show up are the same in how they are presented in the DOM.

So, we could just grab the h4 items in an array specifically like so:
```js
// first, we grab all the `siteContent` keys into an array
const mainContentKeys = Object.keys(siteContent['main-content']);
// now we need to grab the h4 keys specifically - luckily every h4 element ends with the string 'h4'
const mainContentH4s = mainContentKeys.filter(key => key.endsWith('h4'));
// and finally, we can loop through our h4 elements and set them using the equivalent index of our mainContentH4s array
mainContentH4s.forEach((value, idx) => {
  H4[idx] = siteContent['main-content'][value];
});
```